index.html

```diff
@@ -1,15 +1,39 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="style.css">
-    <title>Hello Microverse</title>
+    <title>Portfolio</title>
 </head>
+
 <body>
-    <header>
-        <h1>Hello Microverse</h1>
+    <header class="header_container">
+        <div class="item header_img"></div>
+        <div class="item menu_icon"></div>
```

Comment:
- To enforce proper use of HTML semantics kindly ensure that the hamburger button is on the right and is wrapped with a `<nav>` tag.

Reply:
- [ ] This issue is well resolved :heavy_check_mark: